### PR TITLE
chore: Update README.md for release version 0.3.0

### PR DESCRIPTION
**Pull request by dvandra**
Updating GIFs and Screenshots with latest UI Changes.

https://github.com/dvandra/fabric8-analytics-vscode-extension/tree/update-readme-0.3.0#dependency-analytics

### CODE REVIEW

**Review by arajkumar — CHANGES_REQUESTED** [review]
@dvandra , Please include the changelog  & version updates in the same PR.

**Comment by dvandra** [comment]
> @dvandra , Please include the changelog & version updates in the same PR.

On it!

**Review by dgpatelgit — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,13 +10,18 @@
 - enhancement - Show status bar text based on component analysis status. See [#459](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/459)
     - Notification will be shown only once per manifest in single session.
     - Further changes will be updated only via status bar.
-- enhancement - Updated fabric8-analytics-lsp-server to latest version (v[0.4.15](https://www.npmjs.com/package/fabric8-analytics-lsp-server/v/0.4.15)): See [#459](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/459)
+- enhancement - Updated fabric8-analytics-lsp-server to latest version (v[0.4.19](https://www.npmjs.com/package/fabric8-analytics-lsp-server/v/0.4.19)): See [#469](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/469)
     - Golang CA support from LSP. See [#148](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/148)
     - Handle replace directive from go.mod. See [#162](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/162)
+    - Use concise text for CA popup. See [#472](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/472)
```
Comment (arajkumar) [inline CHANGELOG.md:16]: Concise text is not part of lsp.

```suggestion
    - Use concise text for CA popup. See [#472](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/472)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,13 +10,18 @@
 - enhancement - Show status bar text based on component analysis status. See [#459](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/459)
     - Notification will be shown only once per manifest in single session.
     - Further changes will be updated only via status bar.
-- enhancement - Updated fabric8-analytics-lsp-server to latest version (v[0.4.15](https://www.npmjs.com/package/fabric8-analytics-lsp-server/v/0.4.15)): See [#459](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/459)
+- enhancement - Updated fabric8-analytics-lsp-server to latest version (v[0.4.19](https://www.npmjs.com/package/fabric8-analytics-lsp-server/v/0.4.19)): See [#469](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/469)
     - Golang CA support from LSP. See [#148](https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/148)
```
Comment (arajkumar) [inline CHANGELOG.md:14]: Add the following under lsp,

https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/176
https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/177
https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/172

**Review by arajkumar — CHANGES_REQUESTED** [review]

**Review by arajkumar — APPROVED** [review]
lgtm

**Pull request merged**